OneLogin.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -9,6 +9,15 @@
         "version" : "1.2022062300.0"
       }
     },
+    {
+      "identity" : "app-check",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/google/app-check.git",
+      "state" : {
+        "revision" : "5746b2d35c91c50581590ed97abe4c06b5037274",
+        "version" : "10.18.0"
+      }
+    },
     {
       "identity" : "appauth-ios",
```

Comment:
Just spotted this... There might be a good reason it's here, but was a bit surprised to see it, given we should be wrapping all things app auth with our authentication package?